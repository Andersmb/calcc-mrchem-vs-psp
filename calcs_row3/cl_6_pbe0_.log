Starting job 5688396 on c3-6,c5-[33,40],c10-27,c11-25 at Fri May 20 10:47:32 CEST 2022

Copying cl_6_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/doublets
Copying cl_6_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/doublets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688396      cl_6_pbe0_        180                                             00:38:21      0:0 
5688396.bat+      batch         36        1   00:00:02          0   00:00:02   00:38:21      0:0 
5688396.ext+     extern        180        5   00:00:00          0   00:00:00   00:38:21      0:0 
5688396.0      mrchem.x        180       30   00:37:58         29   01:32:13   00:38:03      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688396                                                                          
5688396.bat+     19280K          0     19280K        0              0          0 
5688396.ext+          0          0          0        0              0          0 
5688396.0      8670060K         29 4097616008        0              3          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688396                                                                                               
5688396.bat+       16.90M               0         16.90M        0.03M                0          0.03M 
5688396.ext+        0.00M               0          0.00M            0                0              0 
5688396.0           7.32M              25          6.96M       67.01M               25         47.68M 

Job 5688396 completed at Fri May 20 11:25:54 CEST 2022
