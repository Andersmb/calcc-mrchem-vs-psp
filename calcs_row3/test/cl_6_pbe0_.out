                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       O            :        0.000000        0.000000       -2.043779
    1      Cl            :        0.000000        0.000000        0.922334
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              13
 Beta electrons          :                                              12
 Total electrons         :                                              25
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999998322342e-01
    1    1    a          :                              9.999999498897e-01
    2    1    a          :                              9.999999731378e-01
    3    1    a          :                              1.000000042931e+00
    4    1    a          :                              1.000000096376e+00
    5    1    a          :                              1.000000002365e+00
    6    1    a          :                              9.999999852325e-01
    7    1    a          :                              9.999995315572e-01
    8    1    a          :                              9.999999895242e-01
    9    1    a          :                              9.999998667066e-01
   10    1    a          :                              9.999998654364e-01
   11    1    a          :                              1.000000392058e+00
   12    1    a          :                              1.000000389965e+00
   13    1    b          :                              9.999998205658e-01
   14    1    b          :                              9.999999770561e-01
   15    1    b          :                              9.999999731367e-01
   16    1    b          :                              1.000000042931e+00
   17    1    b          :                              1.000000096232e+00
   18    1    b          :                              1.000000002366e+00
   19    1    b          :                              9.999999849952e-01
   20    1    b          :                              9.999995372584e-01
   21    1    b          :                              9.999999852619e-01
   22    1    b          :                              9.999998660293e-01
   23    1    b          :                              9.999998655644e-01
   24    1    b          :                              1.000000393696e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                530.748807697561
 E-N energy              :            (au)              -1356.614233284519
 Coulomb energy          :            (au)                284.145793063084
 Exchange energy         :            (au)                 -8.921777313957
 X-C energy              :            (au)                -27.587528723152
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 45.851239745543
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -578.228938560982
 Nuclear energy          :            (au)                 45.851239745543
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.323776988154e+02
                         :      (kcal/mol)             -3.340720470911e+05
                         :        (kJ/mol)             -1.397757445936e+06
                         :            (eV)             -1.448673455450e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -101.206380994455
    1    1    a          :            (au)                -18.991049678612
    2    1    a          :            (au)                 -9.644851250025
    3    1    a          :            (au)                 -7.393000744110
    4    1    a          :            (au)                 -7.387279701350
    5    1    a          :            (au)                 -7.375791178747
    6    1    a          :            (au)                 -1.106990090240
    7    1    a          :            (au)                 -0.802698822570
    8    1    a          :            (au)                 -0.497130034903
    9    1    a          :            (au)                 -0.469078984755
   10    1    a          :            (au)                 -0.422222002398
   11    1    a          :            (au)                 -0.265380512473
   12    1    a          :            (au)                 -0.213920933965
   13    1    b          :            (au)               -101.203520181295
   14    1    b          :            (au)                -18.976118703501
   15    1    b          :            (au)                 -9.642091101571
   16    1    b          :            (au)                 -7.391439266578
   17    1    b          :            (au)                 -7.377354430608
   18    1    b          :            (au)                 -7.374290599842
   19    1    b          :            (au)                 -1.080137555618
   20    1    b          :            (au)                 -0.767660130663
   21    1    b          :            (au)                 -0.461063338087
   22    1    b          :            (au)                 -0.408573729271
   23    1    b          :            (au)                 -0.400044011450
   24    1    b          :            (au)                 -0.197197760704
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -311.055265737789
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          5.000000e+00        -532.377698815439         -5.323777e+02
    1          2.760399e-01        -535.027442750346         -2.649744e+00
    2          1.128114e-01        -535.117202609243         -8.975986e-02
    3          6.871936e-02        -535.132688683787         -1.548607e-02
    4          3.554505e-02        -535.137182880429         -4.494197e-03
    5          4.336490e-02        -535.141444469668         -4.261589e-03
    6          1.462620e-02        -535.142088024022         -6.435544e-04
    7          1.778852e-02        -535.142712951335         -6.249273e-04
    8          9.013299e-03        -535.142837574882         -1.246235e-04
    9          1.337981e-02        -535.142959021858         -1.214470e-04
   10          8.194171e-03        -535.143030694353         -7.167249e-05
   11          1.094935e-02        -535.143105041282         -7.434693e-05
   12          3.828244e-03        -535.143122509594         -1.746831e-05
   13          2.411862e-03        -535.143132461858         -9.952264e-06
   14          3.059218e-03        -535.143141714672         -9.252814e-06
   15          1.687509e-03        -535.143144408253         -2.693580e-06
   16          8.515066e-04        -535.143145606797         -1.198545e-06
   17          4.528589e-04        -535.143146082964         -4.761662e-07
   18          4.014316e-04        -535.143146410279         -3.273155e-07
   19          3.565510e-04        -535.143146575173         -1.648943e-07
   20          3.813434e-04        -535.143146695502         -1.203288e-07
   21          4.005151e-04        -535.143146807399         -1.118970e-07
   22          2.549154e-04        -535.143146863548         -5.614868e-08
   23          1.348306e-04        -535.143146888201         -2.465310e-08
   24          1.028326e-04        -535.143146900505         -1.230399e-08
   25          6.936974e-05        -535.143146906836         -6.330652e-09
---------------------------------------------------------------------------
                      SCF converged in 25 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       O            :        0.000000        0.000000       -2.043779
    1      Cl            :        0.000000        0.000000        0.922334
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                534.320686031794
 E-N energy              :            (au)              -1363.591294091726
 Coulomb energy          :            (au)                284.797416021180
 Exchange energy         :            (au)                 -8.917991875205
 X-C energy              :            (au)                -27.603202738421
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 45.851239745543
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -580.994386652379
 Nuclear energy          :            (au)                 45.851239745543
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.351431469068e+02
                         :      (kcal/mol)             -3.358073919545e+05
                         :        (kJ/mol)             -1.405018128850e+06
                         :            (eV)             -1.456198622735e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -101.799157908470
    1    1    a          :            (au)                -19.310314035243
    2    1    a          :            (au)                 -9.619178609943
    3    1    a          :            (au)                 -7.354082455970
    4    1    a          :            (au)                 -7.350397910562
    5    1    a          :            (au)                 -7.342156821613
    6    1    a          :            (au)                 -1.134615682621
    7    1    a          :            (au)                 -0.819655950542
    8    1    a          :            (au)                 -0.511613358414
    9    1    a          :            (au)                 -0.509025478999
   10    1    a          :            (au)                 -0.469164254199
   11    1    a          :            (au)                 -0.347363332266
   12    1    a          :            (au)                 -0.306991295643
   13    1    b          :            (au)               -101.796851752034
   14    1    b          :            (au)                -19.289235070503
   15    1    b          :            (au)                 -9.617066188694
   16    1    b          :            (au)                 -7.353450496301
   17    1    b          :            (au)                 -7.342047101794
   18    1    b          :            (au)                 -7.340955783310
   19    1    b          :            (au)                 -1.096605431236
   20    1    b          :            (au)                 -0.784356464239
   21    1    b          :            (au)                 -0.492508088986
   22    1    b          :            (au)                 -0.459464756424
   23    1    b          :            (au)                 -0.427152901296
   24    1    b          :            (au)                 -0.293146114810
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -313.166557244112
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000       -0.000006        1.160134
 Magnitude               :            (au)                        1.160134
                         :         (Debye)                        2.948767
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000       -0.670551
 Magnitude               :            (au)                        0.670551
                         :         (Debye)                        1.704370
---------------------------------------------------------------------------
 Total vector            :       -0.000000       -0.000006        0.489584
 Magnitude               :            (au)                        0.489584
                         :         (Debye)                        1.244397
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 38m  0s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
